tracking-migration/build.gradle

```
@@ -0,0 +1,35 @@
+import io.customer.android.Configurations
+
+plugins {
+    id 'com.android.library'
+    id 'kotlin-android'
+    id 'com.twilio.apkscale'
+}
+
+ext {
+    PUBLISH_GROUP_ID = Configurations.artifactGroup
+    PUBLISH_ARTIFACT_ID = "tracking-migration"
+}
+
+apply from: "${rootDir}/scripts/publish-module.gradle"
+apply from: "${rootDir}/scripts/android-config.gradle"
+apply from: "${rootDir}/scripts/codecov-android.gradle"
+apply from: "${rootDir}/scripts/android-module-testing.gradle"
+
+android {
+    namespace 'io.customer.tracking.migration'
+    defaultConfig {
+        consumerProguardFiles "consumer-rules.pro"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+
+dependencies {
+    api project(":base")
```

We'll need core here as well 🤓 
```suggestion
    api project(":base")
    api project(":core")
```